src/converse-omemo.js

```diff
@@ -502,6 +502,7 @@ function getOMEMOToolbarButton (toolbar_el, buttons) {
                 @click=${toggleOMEMO}>
         <converse-icon class="fa ${model.get('omemo_active') ? `fa-lock` : `fa-unlock`}"
                     path-prefix="${api.settings.get('assets_path')}" size="1em"
+                    color="${model.get('omemo_active') ? `var(--info-color)` : `var(--error-color)`}"
```

Comment:
@bernard-ng Did you check whether the color updates dynamically? I.e. when you click the button, does it switch from green to red?

Reply:
Yes it does